### PR TITLE
CSS Colors: fixed issue #3542

### LINKED ISSUE #3542: CSS Colors: Remove vertical divider after 'Show more'
screen shot 2016-08-19 at 9 13 03 am
   

------
IA Page:  http://duck.co/ia/view/css_colors 
[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @sahildua2305

### PR DESCRIPTION
**Pull request by amarlearning**
###### What does your Pull Request do (check all that apply)?
Choose the most relevant items and use the following title template to name
your Pull Request.

- [ ] New Instant Answer
    - [ ] Cheat Sheets: **`New {Cheat Sheet Name} Cheat Sheet`**
    - [ ] Other: **`New {IA Name} Instant Answer`**
- [x] Improvement
    - [x] Bug fix: **`css_colors: Fix Issue #3542`**
    - [ ] Enhancement: **`{IA Name}: {Description of Improvements}`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**

###### Description of changes
What changes does this PR introduce? 
fixed issue #3542

###### Related Issues or Discussions 
What issues or discussions (if any) does this PR address? Please provide the issue # or link.
fixed #3542

###### People to notify
@mention any relevant people/organizations
@moollaza 

---

Instant Answer Page: https://duck.co/ia/view/css_colors

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @sahildua2305

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by amarlearning** [comment]
output : 
screenshot from 2016-10-10 13-49-57

**Comment by moollaza** [comment]
@amarlearning there's no need to add a JS file to specify the `sourceUrl` -- you can actually do that all in the Perl.

You just need to add a `meta` section like we've done [here](https://github.com/duckduckgo/zeroclickinfo-goodies/blob/ba2760f03485f32f10de1e85c19b11415cb8528b/lib/DDG/Goodie/CamelCase.pm#L38).

**Comment by amarlearning** [comment]
@moollaza updated the Perl file & deleted JS file!

File: lib/DDG/Goodie/CssColors.pm [hunk 1]
```diff
@@ -53,6 +53,10 @@ handle remainder => sub {
                 options => {
                     list_content => 'DDH.css_colors.content'
                 }
+            },
+            meta => {
+                sourceName => 'w3schools.com',
+                sourceUrl => 'http://www.w3schools.com/cssref/css_colors.asp'
```
Comment (moollaza) [inline lib/DDG/Goodie/CssColors.pm]: @amarlearning let's use https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#Color_keywords

Seeing as we're using the list of colours recognized by Mozilla 👍

File: lib/DDG/Goodie/CssColors.pm [hunk 2]
```diff
@@ -53,6 +53,10 @@ handle remainder => sub {
                 options => {
                     list_content => 'DDH.css_colors.content'
                 }
+            },
+            meta => {
+                sourceName => 'w3schools.com',
```
Comment (moollaza) [inline lib/DDG/Goodie/CssColors.pm]: We can switch this to `Mozilla Developer Network`

**Comment by amarlearning** [comment]
screenshot from 2016-10-13 01-01-05

**Comment by sahildua2305** [comment]
@moollaza can you please approve the changes you requested for, so that I can go ahead and merge it?

**Pull request merged**